--- gcc-4.7-20120128/gcc/config/linux.h.orig	2012-01-28 23:05:57.118067237 -0200
+++ gcc-4.7-20120128/gcc/config/linux.h	2012-01-28 23:07:30.056532353 -0200
@@ -74,9 +74,9 @@ see the files COPYING3 and COPYING.RUNTI
    GLIBC_DYNAMIC_LINKER must be defined for each target using them, or
    GLIBC_DYNAMIC_LINKER32 and GLIBC_DYNAMIC_LINKER64 for targets
    supporting both 32-bit and 64-bit compilation.  */
-#define UCLIBC_DYNAMIC_LINKER "/lib/ld-uClibc.so.0"
-#define UCLIBC_DYNAMIC_LINKER32 "/lib/ld-uClibc.so.0"
-#define UCLIBC_DYNAMIC_LINKER64 "/lib/ld64-uClibc.so.0"
+#define UCLIBC_DYNAMIC_LINKER "/usr/uclibc/lib/ld-uClibc.so.0"
+#define UCLIBC_DYNAMIC_LINKER32 "/usr/uclibc/lib/ld-uClibc.so.0"
+#define UCLIBC_DYNAMIC_LINKER64 "/usr/uclibc/lib64/ld64-uClibc.so.0"
 #define UCLIBC_DYNAMIC_LINKERX32 "/lib/ldx32-uClibc.so.0"
 #define BIONIC_DYNAMIC_LINKER "/system/bin/linker"
 #define BIONIC_DYNAMIC_LINKER32 "/system/bin/linker"
